conda/core/package_cache_data.py

```diff
@@ -28,9 +28,9 @@
                                   read_index_json_from_tarball, read_repodata_json)
 from ..gateways.disk.test import file_path_is_writable
 from ..models.dist import Dist
-from ..models.index_record import PackageRecord, PackageRef
+from ..models.records import PackageRecord, PackageRef
 from ..models.match_spec import MatchSpec
-from ..models.package_cache_record import PackageCacheRecord
+from conda.models.records import PackageCacheRecord
```

Comment:
Why the mixture of import types??

`conda.models.records` vs `..models.records`